nibabel/fileslice.py

```diff
@@ -767,9 +767,12 @@ def strided_scalar(shape, scalar=0.):
     strided_arr : array
         Array of shape `shape` for which all values == `scalar`, built by
         setting all strides of `strided_arr` to 0, so the scalar is broadcast
-        out to the full array `shape`.
+        out to the full array `shape`. `strided_arr` is flagged as not
+        `writeable`.
```

Comment:
Just something like `We set the array read-only to avoid a numpy error when broadcasting - see https://github.com/numpy/numpy/issues/6491`.